assets/js/blocks/collection-filters/inner-blocks/stock-filter/block.json

```diff
@@ -0,0 +1,46 @@
+{
+	"name": "woocommerce/collection-stock-filter",
+	"version": "1.0.0",
+	"title": "Stock Filter",
+	"description": "Enable customers to filter the product collection by stock status.",
+	"category": "woocommerce",
+	"keywords": [ "WooCommerce", "filter", "stock" ],
+	"supports": {
+		"interactivity": true,
+		"html": false,
+		"multiple": false
+	},
+	"attributes": {
+		"className": {
+			"type": "string",
+			"default": ""
+		},
+		"showCounts": {
+			"type": "boolean",
+			"default": false
+		},
+		"displayStyle": {
+			"type": "string",
+			"default": "list"
+		},
+		"selectType": {
+			"type": "string",
+			"default": "multiple"
+		},
+		"isPreview": {
+			"type": "boolean",
+			"default": false
+		},
+		"queryParam": {
+			"type": "object",
+			"default": {
+				"calculate_stock_status_counts": "true"
+			}
+		}
+	},
+	"usesContext": [ "query", "queryId", "postId", "collectionData" ],
```

Comment:
Did you add these for debugging `"query", "queryId", "postId"`? The filter blocks should only need `collectionData`, theoretically 😄 .

Reply:
This was copypasta from the original, forgot to remove it. Thanks!

Reply:
Fixed it [here](https://github.com/woocommerce/woocommerce-blocks/pull/11663/commits/c57512461915dd3ef9e5de6d9ec9fcac9f501a7c)